core/modules/parser/QSMySqlListener.cc

```diff
@@ -2219,19 +2240,21 @@ class MathExpressionAtomAdapter :
             ASSERT_EXECUTION_CONDITION(false, "Unhandled operatorType.", _ctx);
             break;
 
-        case MathOperatorCBH::SUBTRACT: {
-            bool success = ValueExprFactory::addOp(_getValueExpr(), query::ValueExpr::MINUS);
-            ASSERT_EXECUTION_CONDITION(success,
-                    "Failed to add an operator to valueExpr:" << _getValueExpr(), _ctx);
-            break;
-        }
+        case MathOperatorCBH::SUBTRACT:
+            {
+                bool success = ValueExprFactory::addOp(_getValueExpr(), query::ValueExpr::MINUS);
+                ASSERT_EXECUTION_CONDITION(success,
+                        "Failed to add an operator to valueExpr:" << _getValueExpr(), _ctx);
+                break;
+            }
```

Comment:
Why change the style? I think previous was more consistent with LSST style where opening brace goes on the same line.